<launch>
    <arg name="model" default="$(find livox_laser_simulation)/urdf/hans_bot.urdf.xacro"/>


    <include file="$(find livox_laser_simulation)/launch/gazebo.launch">
      <arg name="model" value="$(arg model)" />
    </include>
    

  
    <rosparam command="load"
              file="$(find livox_laser_simulation)/config/joints.yaml"
              ns="hans_bot_joint_state_controller" />

    <rosparam command="load"
              file="$(find livox_laser_simulation)/config/h_slider.yaml"
              ns="hans_bot_slider_controller" />
              
    <rosparam command="load"
              file="$(find livox_laser_simulation)/config/h_head.yaml"
              ns="hans_bot_head_controller" 
              />
  
    <rosparam command="load"
              file="$(find livox_laser_simulation)/config/h_wheels.yaml"
              ns="hans_bot_wheel_controller" />

    <node name="hans_bot_controller_spawner" pkg="controller_manager" type="spawner"
      args="hans_bot_joint_state_controller
            hans_bot_slider_controller
            hans_bot_head_controller
            hans_bot_wheel_controller
            --shutdown-timeout 10"/>
            

    

    
           




            
</launch>
